src/libs/ajax.js

```diff
@@ -1102,6 +1105,33 @@ const DataRepo = signal => ({
   }
 })
 
+const AzureStorage = signal => ({
+  details: async (workspaceId = {}) => {
+    const res = await fetchWorkspaceManager(`workspaces/v1/${workspaceId}/resources?stewardship=CONTROLLED`,
```

Comment:
There is an optional parameter to request just a resources of a specific type, but it does not allow multiple resource types to be specified. Therefore I would have to make 2 different calls if I wanted to filter by resource type, and that does not seem worth the complexity.